tests/cacl/test_cacl_application.py

```diff
@@ -59,7 +63,128 @@ def docker_network(duthost):
 
     return docker_network
 
+@pytest.fixture(scope="function")
+def collect_ignored_rules(duthosts, rand_one_dut_hostname):
+    """
+    Collect existing iptables rules before test, set them as ignored as they are not related to CACL test cases.
+
+    Args:
+        duthosts: All DUTs belong to the testbed.
+        rand_one_dut_hostname: hostname of a random chosen dut to run test.
+
+    Returns:
+        None
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    # clear other control ACL rules before test to avoid miscalucation
+    duthost.command("acl-loader delete SNMP_ACL")
+    duthost.command("acl-loader delete NTP_ACL")
+    duthost.command("acl-loader delete SSH_ONLY")
```

Comment:
I think it's better not to delete the rules before testing. Just add them into the collected ignore list.

Reply:
Okay.

Reply:
Done